src/main/java/net/greatstart/dao/hibernate/UserDaoImpl.java

```diff
@@ -36,3 +42,4 @@ public User getById(long id) {
         return null;

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1168)